client/.env

```diff
@@ -1 +1,4 @@
 EXTEND_ESLINT=true
+REACT_APP_SENDGRID_CONTACS_API_URL=TODO
+REACT_APP_SENDGRID_API_KEY=
+REACT_APP_SENDGRID_LIST_ID=
```

Comment:
Please let me know if you need help finding these values for testing or adding them to the environmental variables.

Reply:
Do you recall why we had to check this in? I don't think we'll want to check these REACT_APP... ones if this is the case I think we'll need to handle these similarly to GTM env vars by adding via GitHub Action Workflows. 

Most devs won't need it, so we just want to make sure it does nothing if the env var isn't set.

```
       env:
          GTM_ID: ${{ secrets.PRODUCTION_GTM_ID }}
        run: >
          echo "REACT_APP_GTM_ID=${GTM_ID}" > ./client/.env
```

Reply:
Removed :)